lib/config/presets/internal/custom-managers.ts

```diff
@@ -4,8 +4,7 @@ import type { Preset } from '../types';
 
 export const presets: Record<string, Preset> = {
   dockerfileVersions: {
-    description: 'Update `_VERSION` variables in Dockerfiles.',
```

Comment:
This seems like it wasn't copy pasted new?

Reply:
Still working on a bad merge.